tests/Utils/ShopUserLoginTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Api\Utils;
```

Comment:
This trait should be moved to the `Api` directory according to the namespace

Reply:
This is analogical to what we have on open source 🤔
`Sylius\Tests\Api\Utils\ShopUserLoginTrait`

Reply:
What is even more `ShopUserLoginTrait` is not used at all in my code haha, I'll remove it in a separate PR.